Libs/MRML/Core/Testing/vtkMRMLTransformNodeTest1.cxx

```diff
@@ -10,10 +10,43 @@
 
 =========================================================================auto=*/
 
+#include "vtkMRMLBSplineTransformNode.h"
 #include "vtkMRMLCoreTestingMacros.h"
+#include "vtkMRMLScene.h"
 #include "vtkMRMLTransformNode.h"
 
 #include <vtkGeneralTransform.h>
+#include <vtkMatrix4x4.h>
+#include <vtkNew.h>
+#include <vtkTransform.h>
+
+bool Matrix4x4AreEqual(vtkMatrix4x4 *m1, vtkMatrix4x4 *m2)
```

Comment:
Would it make sense to re-use `vtkMRMLSliceNode::Matrix4x4AreEqual()` ?
See https://github.com/Slicer/Slicer/blob/482657adbb55333e4f97ccb4eed525e16e6a3d06/Libs/MRML/Core/vtkMRMLSliceNode.h#L231